recipes/gradsflow/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "gradsflow" %}
+{% set version = "0.0.7.post2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 04ae44f963e91503ba235254eff96bfee65ec3ed6eb340d2e53e5fd8ca3d2729
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - flit-core >=3.2,<4
+  run:
+    - python >=3.7
+    - accelerate >=0.5.0
+    # - lightning-flash >=0.5.1
+    - loguru >=0.5    
+    - ray-default >=1.6.0
+    - ray-tune >=1.6.0
+    - rich
+    - smart_open >=5.1    
+    - timm
+    - pytorch >=1.4
+    - torchvision
+    - torchmetrics >=0.6.0
+  run_constrained:
+    - comet_ml
+
+test:
+  imports:
+    - {{ name|lower }}
```

Go ahead and explicitly list the name here. If the package fully changes the import, odds are we want a new package anyway.
```suggestion
    - gradsflow
```

---

Done.